src/gemdat/plots/_vibration.py

```diff
@@ -91,3 +94,76 @@ def vibrational_amplitudes(*, trajectory: Trajectory) -> plt.Figure:
            ylabel='Occurrence (a.u.)')
 
     return fig
+
+
+def vibrational_amplitudes2(*,
+                            trajectory: Trajectory,
+                            bins: int = 50,
+                            n_parts: int = 1) -> go.Figure:
+    """Plot histogram of vibrational amplitudes with fitted Gaussian.
+
+    Parameters
+    ----------
+    trajectory : Trajectory
+        Input trajectory, i.e. for the diffusing atom
+    n_parts : int
+        Number of parts for error analysis
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        Output figure
+    """
+
+    trajectories = trajectory.split(n_parts)
+    single_metrics = SimulationMetrics(trajectory)
+    metrics = [
+        SimulationMetrics(trajectory).amplitudes()
+        for trajectory in trajectories
+    ]
+
+    maxamp = max(map(lambda metric: max(metric), metrics))
+    minamp = min(map(lambda metric: min(metric), metrics))
+    maxamp = max(abs(minamp), maxamp)
+    minamp = -maxamp
```

Comment:
🤯

Would it be possible to refactor this without lambda and make it easier to read?